examples/odata-cli/.eslintrc

```diff
@@ -1,5 +1,6 @@
 {
     "extends": ["../../.eslintrc"],
+    "parserOptions": { "project": "./tsconfig.esbuild.json" },
```

Comment:
```suggestion
    "parserOptions": { "project": "./tsconfig.eslint.json" },
```
As discussed please rename the config file

Reply:
Thanks, @devinea for catching this typo. I renamed the files.